odoo/addons/base/res/res_config.py

```diff
@@ -536,6 +584,25 @@ def execute(self):
                 groups.write({'implied_ids': [(3, implied_group.id)]})
                 implied_group.write({'users': [(3, user.id) for user in groups.mapped('users')]})
 
+        # config fields: store ir.config_parameters
+        SuIrConfigParameter = self.env['ir.config_parameter'].sudo()
+        for name, icp in classified['config']:
+            value = self[name]
+            if value and self._fields[name].type == 'char':
+                # storing developer keys as ir.config_parameter may lead to nasty
+                # bugs when users leave spaces around them
+                value = value.strip()
+            elif value and self._fields[name].type in ['integer', 'float']:
+                value = repr(value)
+            elif value and self._fields[name].type == 'many2one':
+                value = value.id
+            else:
+                # /!\ tricky:
+                # - value can be falsy in case of a many2one but being an empty recordset
+                # - we actually want to send the boolean False to set_param
+                value = False
```

Comment:
Hey, boolean and selection fields are always forced to `False`!

Reply:
When fixing an issue causes an other one ;) I will check that, thanks for pointing it out